lms/static/sass/discussion/_discussion.scss

```diff
@@ -169,8 +157,8 @@ body.discussion {
     display: inline-block;
     width: 20px;
     height: 20px;
-    background-image: url('/static/images/wmd-buttons-transparent.png');
-    background-position: 0px 0px;
+    background-image: url('#{$static-path}/images/wmd-buttons-transparent.png');
+    background-position: 0 0;
```

Comment:
The default value for `background-position` is `0% 0%` so I think you might be able to lose this line completely?